TESTING.md

```diff
@@ -466,6 +468,7 @@ To open the context menu:
 1. Login to Cody Free account
 2. Create new chat
 3. User doesn't see model dropdown
```

Comment:
I think it is not true for a while already. We do display the dropdown with CTA for the Pro items. If I am right, you can fix that btw

Reply:
Nice catch, updated!